package.json

```diff
@@ -31,28 +31,29 @@
     "main": "can-stache-bindings"
   },
   "dependencies": {
+    "can-attribute-encoder": "^0.3.0",
```

Comment:
If the API changes, we should publish `0.4.0`. In that case, it wouldn't get updated here.

Not hard-coding it allows us to publish bug fixes and doc updates.